pkg/client/cmd/team.go

```diff
@@ -45,12 +45,25 @@ You need to create a user before use this command.`,
 	Run: teamAddUser,
 }
 
+var teamRemoveUserCmd = &cobra.Command{
+	Use:   "remove-user",
+	Short: "Remove a member of a team",
+	Long: `Remove a member of a team.
+
+You can remove an user of a team with:
+
+  $ teresa team remove-user --user john.doe@foodomain.com --team foo
```

Comment:
Maybe user can be an arg instead of a flag

Reply:
I think that for now we need consistency with `add-user` but it is an interesting option for another PR.

Reply:
Nice :)